tools/api_protolock/BUILD

```diff
@@ -0,0 +1,47 @@
+load("@rules_python//python:defs.bzl", "py_test")
+
+licenses(["notice"])  # Apache 2
+
+py_test(
+    name = "protolock_test",
+    srcs = ["protolock_test.py"],
+    data = [
+        "//tools/testdata/api_protolock:allowed/test_add_comment_current",
+        "//tools/testdata/api_protolock:allowed/test_add_comment_next",
+        "//tools/testdata/api_protolock:allowed/test_add_enum_value_current",
```

Comment:
Out of curiosity, how come some equivalent of these tests doesn't exist in Protolock upstream? Would these belong there at some point?